.github/workflows/ci.yml

```diff
@@ -2,6 +2,9 @@ name: CI/CD
 
 on:
   push:
+    branches:
```

Comment:
One not-quite-related change - the tests currently were running twice when I make a pull request from a feature branch.